src/cljs/time_tracker_web_nxt/subs.cljs

```diff
@@ -23,9 +23,60 @@
 (create-subscription :show-create-timer-widget?)
 (create-subscription :show-user-menu?)
 (create-subscription :clients)
-(create-subscription :client) ;; TODO: check if we are using this subscription anywhere
-(create-subscription :selected-client)
-(create-subscription :selected-project)
-(create-subscription :selected-task)
-(create-subscription :projects)
-(create-subscription :tasks)
+(create-subscription :client)
+
+(rf/reg-sub
+ :all-projects
+ (fn [db]
+   (:projects db)))
+
+(rf/reg-sub
+ :projects
```

Comment:
Why not keep the naming consistent with the existing ones (ex: timers), and rename this to be `projects-for-client` instead?